src/main.cpp

```diff
@@ -135,9 +135,10 @@ static inline void disable_all_plugins(bool* plugin_list)
         plugin_list[i] = false;
 }
 
-static inline void enable_plugin(char* optarg, bool* plugin_list, bool &disabled_all)
+static inline void enable_plugin(char* optarg, bool* plugin_list, bool& disabled_all)
```

Comment:
This doesn't look right, this should be `bool *disabled_all`

Reply:
fixed!